src/ralph/models/edx/drag_and_drop/fields/events.py

```diff
@@ -0,0 +1,60 @@
+"""Video event fields definitions."""
+
+from typing import Optional
+
+from ...base import AbstractBaseEventField
+
+
+class EdxDragAndDropV2FeedbackEventField(AbstractBaseEventField):
+    """Pydantic model for drag and drop feedback statements `event` field.
+
+    Attributes:
+        content (str): Consists of the text of the success or error feedback
+            message in the pop up.
+        manually (bool): Set to `true` when the learner manually closed the pop up
+        dialog box, `false` when the browser closed it.
+        truncated (bool): Set to `true` if the content was longer than 12500 characters.
+    """
+
+    content: str
+    manually: bool
+    truncated: Optional[bool]
+
+
+class EdxDragAndDropV2ItemEventField(AbstractBaseEventField):
+    """Pydantic model for drag and drog item statements `event` field.
+
+    Attributes:
+        item_id (str): Consists of the index assigned to the draggable item
+            selected by the learner.
+    """
+
+    item_id: str
```

Comment:
I find this to be more confusing than having two `ItemPickedUp` and `ItemDropped` fields, which would both have an `item_id`. WDYT?

Reply:
On another note, wouldn't `item_id` always be an integer instead of a string?